.ci.yaml

```diff
@@ -36,6 +36,20 @@ targets:
           {"dependency": "vs_build"}
         ]
 
+  - name: Windows unit tests master - packages
+    recipe: packages/packages
+    timeout: 30
+    bringup: true
+    properties:
+      add_recipes_cq: "true"
+      target_file: windows_unit_tests.yaml
```

Comment:
windows_dart_unit_test.yaml

(There are native unit tests in the plugins repo, so "unit tests" is ambiguous in our CI.)